schema/me/conversation/index.js

```diff
@@ -144,16 +158,29 @@ export const ConversationFields = {
 
   artworks: {
     type: new GraphQLList(ArtworkType),
-    resolve: conversation => {
-      const ids = []
-
-      for (const item of conversation.items) {
+    description: "Only the artworks discussed in the conversation.",
+    resolve: ({ items }) => {
+      const results = []
+      items.forEach(item => {
         if (item.item_type === "Artwork") {
-          ids.push(item.item_id)
+          results.push(item.properties)
         }
-      }
+      })
+      return results
+    },
+  },
 
-      return gravity("artworks", { ids })
+  items: {
+    type: new GraphQLList(ConversationItemType),
+    description: "The artworks and/or partner shows discussed in the conversation.",
+    resolve: ({ items }) => {
+      const results = []
+      items.forEach(item => {
+        if (item.item_type === "Artwork" || item.item_type === "PartnerShow") {
+          results.push(item.properties)
+        }
+      })
+      return results
```

Comment:
I think this can just be `({ items }) => items.map(({ properties }) => properties))`, no? (I assume `items` will be an empty list if we weren’t able to backfill the data?)